content/01.abstract.md

```diff
@@ -1,3 +1,9 @@
 ## Abstract {.page_break_before}
 
-
+Networks of biomedical data are subject to both false positives and false negatives, meaning that entities rarely have all their true connections represented in any given network.
+Deviations from true connections can be biased with respect to node degree due to a variety of processes including inspection bias and experimental methods.
+While degree is subject to potentially substantial biases, link prediction methods can be strongly affected by degree.
+In the present work, we introduce a network permutation framework to quantify the effect of node degree on network-based methods and prediction tasks.
+We introduce the "edge prior", a permutation-based feature that quantifies a node pair's affinity to share an edge across permuted networks.
+After demonstrating that this prior feature shows excellent discrimination and calibration performance for some prediction tasks, we conclude that our prior feature represents a suitable baseline for network link prediction tasks, as performance exceeding the baseline is attributable to factors other than degree alone.
```

Comment:
"some prediction tasks" is WEAK. Can we say some like "XX different networks (XX bipartite, XX directed, XX undirected)." This would allow us to mention the different network types supported by our xswap implementation.